frontend/packages/console-app/src/components/nav/NavHeader.tsx

```diff
@@ -135,48 +133,40 @@ const NavHeader: React.FC<NavHeaderProps> = ({ onPerspectiveSelected }) => {
       : []),
   ];
 
-  return (
-    <>
-      {activePerspective !== ACM_PERSPECTIVE_ID && (
-        <div
-          className="oc-nav-header"
-          data-tour-id="tour-perspective-dropdown"
-          data-quickstart-id="qs-perspective-switcher"
-        >
-          <Select
-            isOpen={isPerspectiveDropdownOpen}
-            data-test-id="perspective-switcher-menu"
-            onOpenChange={(open) => setPerspectiveDropdownOpen(open)}
-            toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
-              <MenuToggle
-                isFullWidth
-                data-test-id="perspective-switcher-toggle"
-                isExpanded={isPerspectiveDropdownOpen}
-                ref={toggleRef}
-                onClick={() => (perspectiveItems.length === 1 ? null : togglePerspectiveOpen())}
-                className={cx({
-                  'oc-nav-header__menu-toggle--is-empty': perspectiveItems.length === 1,
-                })}
-                icon={<LazyIcon />}
-              >
-                {name && (
-                  <Title headingLevel="h2" size="md">
-                    {name}
-                  </Title>
-                )}
-              </MenuToggle>
-            )}
-            popperProps={{
-              appendTo: () =>
-                document.querySelector("[data-test-id='perspective-switcher-toggle']"),
-            }}
+  return activePerspective !== ACM_PERSPECTIVE_ID && perspectiveItems.length > 1 ? (
```

Comment:
Does this logic work properly with ACM since we handle that perspective specially? If it's just the admin and acm perspectives, is the dropdown still hidden?

cc @KevinFCormier

Reply:
Good question. FWIW, I have advocated for the removal of the special treatment of the "All Clusters" perspective since it causes confusion frequently when people expect the switcher to list all managed clusters, and further, ACM is introducing the ability to refer to "local-cluster" by any name the user chooses. Unsure what we would call this perspective though.

Reply:
Let's re-open this discussion since we're changing perspectives anyway... The cluster dropdown seems to create confusion, and we don't have any near term plans to add other clusters to the list. Can we make ACM a normal perspective? And what should it be called? cc @imjoyjean @kevinhatchoua 

@logonoff Let's update the logic in this PR to handle ACM properly and make any ACM perspective changes in a follow-up. You should filter out the ACM perspective from the array before checking the length since ACM doesn't show up in this dropdown.

Reply:
@spadgett FYI @kevinhatchoua sorry, i was on PTO for a while. Back in the day I think I suggested calling the ACM view "All clusters" - is that the naming that is causing confusion?